### PR TITLE
67 try ingesting branching routes as multilinestring objects

### PR DESCRIPTION
**Pull request by mbjackson-capp**
Ingesting branching routes as MultiLineString objects was successful. Ingestion script for TransitStop and TransitRoute is in good enough condition to merge into main.

### CODE REVIEW

**Review by jimenasalinas — APPROVED** [review]
The code looks good to me! Thank you for resolving yesterday's issue

**Comment by jimenasalinas** [comment]
For future iterations (not urgent) we may want to separate ingestion from extraction to allow for tests to work

**Comment by meganhmoore** [comment]
Can we get a screenshot of tests passing if its too much to migrate tests to the `route_rangers_api/tests.py`

File: app/scripts/extract_scheduled_gtfs.py [hunk 1]
```diff
@@ -146,70 +147,109 @@ def ingest_transit_stations(
...
-            mode=transit_mode,
+            mode=assign_mode(row, url),
         )
         print(
             f'Observation created: {row["city"]}, {row["stop_id"]}, '
             f'{row["stop_name"]}, Point({row["stop_lat"]},{row["stop_lon"]})'
             f", mode {obs.mode}"
         )
-        print(obs)
         try:
```
Comment (JPMartinezClaeys) [inline app/scripts/extract_scheduled_gtfs.py:157]: Good for this stage, but let's keep in mind for later to get more specific try-excepts

File: app/scripts/extract_scheduled_gtfs.py [hunk 2]
```diff
@@ -146,70 +147,109 @@ def ingest_transit_stations(
...
+            return LIGHT_RAIL
+        elif "WES" in row["stop_name"]:
+            return RAIL
+        elif "Tram Terminal" in row["stop_name"]:
+            return AERIAL_LIFT  # there's an aerial tram!!
+        else:
+            return BUS
+
+
+def handroll_multiline_routes(city: str, feed) -> gpd.GeoDataFrame:
```
Comment (JPMartinezClaeys) [inline app/scripts/extract_scheduled_gtfs.py:205]: I get this is fine for now, but I don't find the name of the function particularly clear on what should do.

**Review by JPMartinezClaeys — APPROVED** [review]
This looks good to me

**Comment by meganhmoore** [comment]
Looks good to me! I made some updates with my latest PR that migrated your test file to the `app/route_rangers_api/tests.py` file so that when the imports tried to call these functions that rely on django models they wouldn't immediately error. 

That being said, most of those tests are currently being skipped and outdated imports were commented out (https://github.com/uchicago-capp-30320/RouteRangers/blob/main/app/route_rangers_api/tests.py#L11-L16) so if you want to make an issue to go back and update those tests, delete the ones that are no longer needed, etc. please do! doesn't have to be high priority though

**Review by meganhmoore — APPROVED** [review]

**Pull request merged**